app/controllers/application_controller.rb

```diff
@@ -10,4 +10,10 @@ def configure_permitted_parameters
   end
 
   protect_from_forgery with: :null_session
+
+  rescue_from ActiveRecord::RecordNotFound, with: :handle_record_not_found
+
+  def handle_record_not_found
+    render(json: { error: 'Nothing found' }, status: 404)
```

Comment:
we can use status code symbols here to make it more readable. For 404 it's `:not_found`. You can find a list here http://billpatrianakos.me/blog/2013/10/13/list-of-rails-status-code-symbols/

Reply:
Greate, I thought they existed only in specs